app/v4/createAmendCumulativePeriodSummary/CreateAmendCumulativePeriodSummaryService.scala

```diff
@@ -62,15 +53,16 @@ class CreateAmendCumulativePeriodSummaryService @Inject() (connector: CreateAmen
       "END_DATE_NOT_ALIGNED"               -> RuleEndDateNotAlignedWithReportingTypeError,
       "MISSING_SUBMISSION_DATES"           -> RuleMissingPeriodSummaryDatesError,
       "START_END_DATE_NOT_ACCEPTED"        -> RuleStartAndEndDateNotAllowedError,
-      "OUTSIDE_AMENDMENT_WINDOW"           -> RuleOutsideAmendmentWindowError
+      "OUTSIDE_AMENDMENT_WINDOW"           -> RuleOutsideAmendmentWindowError,
+      "SERVER_ERROR"                       -> InternalError,
+      "SERVICE_UNAVAILABLE"                -> InternalError
     )
 
     val extraTysErrors = Map(
```

Comment:
remove these 3 lines and put the errors below in a suitable place before the API specific errors